02_configure_host.sh

```diff
@@ -15,6 +15,12 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -e "non_root_user=$USER" \
     -e "working_dir=$WORKING_DIR" \
     -e "roles_path=$PWD/roles" \
+    -e "openshift_master_memory=${MASTER_MEMORY}" \
+    -e "openshift_master_vcpu=${MASTER_VCPU}" \
+    -e "openshift_master_disk=${MASTER_DISK}" \
+    -e "openshift_worker_memory=${WORKER_MEMORY}" \
+    -e "openshift_worker_vcpu=${WORKER_VCPU}" \
+    -e "openshift_worker_disk=${WORKER_DISK}" \
```

Comment:
As I mentioned in my previous comment, would it perhaps be better to add a variable that overrides the  tripleo-quickstart-config/metalkube-nodes.yml file, and avoid putting these variables into the example config?

I'm just thinking if we want folks to use the defaults in most cases, we might want to avoid exposing the variables in the example config file, and if folks are changing this they may want to do stuff like modify the number/size of extra disks etc, which is all controlled via  tripleo-quickstart-config/metalkube-nodes.yml

Reply:
Ok, I'll fix it. Thanks!